docsrc/imap/download/release-notes/3.0/index.rst

```diff
@@ -8,7 +8,6 @@ Cyrus IMAP 3.0 Releases
     :maxdepth: 1
     :glob:
 
-    x/?.?.*-beta*
-    x/?.?.*-rc*
-    x/?.?.?
-    x/?.?.??
+    x/*-beta*
+    x/*-rc*
+    x/*
```

Comment:
Does newer sphinx order these sensibly?  sphinx 3.x sorts globs in ascii order, which leads to releases being listed like:
* 3.x.1
* 3.x.10
* 3.x.2

The fiddlier listings here are meant to force the releases to be listed in release order, and we (I) ignore the warnings for the empty two-digit expansions on branches that haven't had a two-digit minor release yet.

Reply:
Discussion on Slack suggested that these are still sorting as we'd like.

Also, I said that I'd like to do some future work to replace the globbing with exact links.  Then, since we'll normally be warning free, we can notice warnings that arise from "document not included".